RELEASE-NOTES.md

```diff
@@ -7,17 +7,32 @@ The format of this file is based on [Keep a Changelog](http://keepachangelog.com
 
 The Axom project release numbers follow [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased] - Release date yyyy-mm-dd
+## [Version 0.3.2] - Release date 2019-09-22
 
 ### Added
+- Added support in Mint for reading and writing an unstructured mesh in the [SU2 Mesh file format].
+  This includes support for both single and mixed cell type topology unstructured mesh types.   
+- Added a new option to enable/disable use of CUB, AXOM_USE_CUB, which is disabled by default. This
+  allows to disable CUB to circumvent issues encountered with the device linker.
```

Comment:
@davidgunderman / @kennyweiss there should probably be a bullet here regarding the new BezierCurve class in `primal`.  If you don't mind, could one of you make a commit on this branch to add a short description about this?

Reply:
I'll add it.

Reply:
Done.